src/views/Maintenancepage.vue

```diff
@@ -0,0 +1,81 @@
zR2LTMuODlhNy40NiA3LjQ2IDAgMDE3LjQ2LTcuNDZ6IiBjbGFzcz0iY2xzLTExIi8+CiAgPHJlY3Qgd2lkdGg9IjE1NS45MyIgaGVpZ2h0PSIyMzAuNTgiIHg9IjE3NC4yNSIgeT0iMTIwLjM4IiBjbGFzcz0iY2xzLTExIiByeD0iMTQuODciIHRyYW5zZm9ybT0icm90YXRlKDkwIDI2MC4wMDUgMTY5Ljg3NSkiLz4KICA8cmVjdCB3aWR0aD0iMjEwLjU4IiBoZWlnaHQ9IjEzNS45MyIgeD0iODguOTMiIHk9Ijk0LjEyIiBjbGFzcz0iY2xzLTIiIHJ4PSI0Ljg3Ii8+CiAgPHBhdGggZD0iTTM1Ny41MSAxNzIuNTd2MTguMjdIMTQ2Ljkzdi0xOC4yN2E0Ljg3IDQuODcgMCAwMTQuODctNC44N2gyMDAuODRhNC44NyA0Ljg3IDAgMDE0Ljg3IDQuODd6IiBjbGFzcz0iY2xzLTEyIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtNTggLTczLjU5KSIvPgogIDxwYXRoIGQ9Ik0zNDMgMTc4LjQ4YTMuMjQgMy4yNCAwIDEwMy4yMy0zLjI0IDMuMjMgMy4yMyAwIDAwLTMuMjMgMy4yNHpNMzI5Ljc5IDE3OC40OGEzLjI0IDMuMjQgMCAxMDMuMjQtMy4yNCAzLjI0IDMuMjQgMCAwMC0zLjI0IDMuMjR6TTMxNS45MyAxNzguNDhhMy4yNCAzLjI0IDAgMTAzLjIzLTMuMjQgMy4yMyAzLjIzIDAgMDAtMy4yMyAzLjI0eiIgY2xhc3M9ImNscy00IiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtNTggLTczLjU5KSIvPgogIDxwYXRoIGQ9Ik0zMzkuMTEgMjY1LjI2SDE2My44YTIuNjYgMi42NiAwIDAxLTIuNjUtMi42NiAyLjY1IDIuNjUgMCAwMTIuNjUtMi42aDE3NS4zMWEyLjY2IDIuNjYgMCAwMTIuNjYgMi42NSAyLjY2IDIuNjYgMCAwMS0yLjY2IDIuNjF6TTMzOS4xMSAyNzguNThIMTYzLjhhMi42NiAyLjY2IDAgMDEtMi42NS0yLjY2IDIuNjYgMi42NiAwIDAxMi42NS0yLjY2aDE3NS4zMWEyLjY2IDIuNjYgMCAwMTIuNjYgMi42NiAyLjY2IDIuNjYgMCAwMS0yLjY2IDIuNjZ6TTIwMS41NSAyOTEuODlIMTYzLjhhMi42NiAyLjY2IDAgMDEtMi42NS0yLjY2IDIuNjYgMi42NiAwIDAxMi42NS0yLjY2aDM3Ljc1YTIuNjYgMi42NiAwIDAxMi42NiAyLjY2IDIuNjYgMi42NiAwIDAxLTIuNjYgMi42NnpNMjY5LjkyIDI5MS44OWgtMzcuNzVhMi42NiAyLjY2IDAgMDEtMi42Ni0yLjY2IDIuNjYgMi42NiAwIDAxMi42Ni0yLjY2aDM3Ljc1YTIuNjYgMi42NiAwIDAxMi42NSAyLjY2IDIuNjYgMi42NiAwIDAxLTIuNjUgMi42NnpNMzM4LjI4IDI5MS44OWgtMzcuNzVhMi42NiAyLjY2IDAgMDEtMi42Ni0yLjY2IDIuNjYgMi42NiAwIDAxMi42Ni0yLjY2aDM3Ljc1YTIuNjYgMi42NiAwIDAxMi42NiAyLjY2IDIuNjYgMi42NiAwIDAxLTIuNjYgMi42NnoiIGNsYXNzPSJjbHMtMTIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTM5NS4xOCAxMjAuMzZBMTAuMTkgMTAuMTkgMCAwMDM4NSAxMTAuMThoLTFhMjQuNzMgMjQuNzMgMCAwMC40Ni00Ljc0IDI1LjA1IDI1LjA1IDAgMDAtNDkuNTYtNS4yIDE1LjU4IDE1LjU4IDAgMTAtNS43NyAzMC4yM2g1NmExMC4xOCAxMC4xOCAwIDAwMTAuMDUtMTAuMTF6IiBjbGFzcz0iY2xzLTMiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTM0Ny41MSAxMDMuNjFhLjU1LjU1IDAgMDEuOTUgMGwzLjA3IDUuMzMgMy4wOCA1LjMzYS41NS41NSAwIDAxLS40Ny44MmgtMTIuMzFhLjU1LjU1IDAgMDEtLjQ4LS44MmwzLjA4LTUuMzN6IiBjbGFzcz0iY2xzLTIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHBhdGggZD0iTTI4Ny43OSAzOS42MWg0LjcxdjI1LjAzaC00LjcxeiIgY2xhc3M9ImNscy0yIi8+CiAgPHBhdGggZD0iTTMwNi4xNCAyOS43NWg0LjcxdjI1LjAzaC00LjcxeiIgY2xhc3M9ImNscy00Ii8+CiAgPHBhdGggZD0iTTM2NS44NiAxMzhhLjU0LjU0IDAgMDAuOTQgMGwzLjA4LTUuMzMgMy4wOC01LjMzYS41NC41NCAwIDAwLS40Ny0uODJoLTEyLjMxYS41NS41NSAwIDAwLS40OC44MmwzLjA4IDUuMzN6IiBjbGFzcz0iY2xzLTQiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC01OCAtNzMuNTkpIi8+CiAgPHRleHQgZmlsbD0iIzMwMzg0MSIgZm9udC1mYW1pbHk9Ik9wZW5TYW5zLVNlbWlCb2xkLE9wZW4gU2FucyIgZm9udC1zaXplPSI0OCIgZm9udC13ZWlnaHQ9IjYwMCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTMzLjQxIDE2OS4zKSI+CiAgICBQR0FKPHRzcGFuIHg9IjExMC40NiIgeT0iMCIgZm9udC1zaXplPSIxMiI+PC90c3Bhbj4KICA8L3RleHQ+Cjwvc3ZnPgo="
```

Comment:
@gonzalo-plaza Now we can add this image on assets/img, the base 64 is only to static page, please add this iamge in this dir and use this image